ironfish-cli/src/commands/service/bridge/relay.ts

```diff
@@ -154,13 +154,45 @@ export default class BridgeRelay extends IronfishCommand {
   ): Promise<void> {
     Assert.isNotUndefined(response)
 
+    const { requests: pendingBurnRequests } = await api.getBridgePendingBurnRequests()
+
+    const pendingBurnTransactions: Map<string, number[]> = new Map()
+    for (const request of pendingBurnRequests) {
+      Assert.isNotUndefined(request.source_burn_transaction)
+
+      const requestIds = pendingBurnTransactions.get(request.source_burn_transaction) ?? []
+
+      requestIds.push(request.id)
+      pendingBurnTransactions.set(request.source_burn_transaction, requestIds)
+    }
+
     const sends = []
     const burns = []
-    const confirms = []
+    const confirmedReleases = []
+    const confirmedBurns = []
 
     const transactions = response.transactions
 
     for (const transaction of transactions) {
+      if (pendingBurnTransactions.has(transaction.hash)) {
+        const requestIds = pendingBurnTransactions.get(transaction.hash) ?? []
+
+        this.log(
+          `Confirmed burn for request IDs [${requestIds.toString()}] in transaction ${
+            transaction.hash
+          }`,
+        )
+
+        for (const requestId of requestIds) {
+          confirmedBurns.push({
+            id: requestId,
```

Comment:
This seems like the right status to me, just wanted you to double check.

Reply:
I think the status is right, but now that you mention it, we might need to do more here than just update the status to prompt the API to do the release on the other side

Reply:
updated to use the `release` endpoint